dovpanda/core.py

```diff
@@ -206,3 +206,20 @@ def dont_append_with_loop(arguments):
         ledger.tell('dont append or concat dfs iteratively. '
                     'it is a better practice to first create a list of dfs. '
                     'and then <code>pd.concat(list_of_dfs)</code> in one go')
+
+
+@ledger.add_hint('Series.str.split', 'post')
+def suggest_expand(res, arguments):
+    expand = arguments.get('expand')
+    pat = arguments.get('pat')
+    if expand:
+        return
+    if hasattr(res, 'name'):
```

Comment:
I'm not sure what you want to check here..if that for the case of series by itself the `name` property will still exists but with Null value, no? i think this condition will never be false

Reply:
I'm trying to be defensive. Maybe `res` returns as some weird datatype on some weird edge cases.

Reply:
So it won't be a series, please change the col name to something more generic than series. Maybe a `structure`?

Reply:
column seems fitting as the user's main use case is to do the op on column